t/api/05-machines.t

```diff
@@ -84,7 +84,11 @@ is_deeply(
 
 
 my $res = $t->post_ok('/api/v1/machines', form => {name => "testmachine"})->status_is(400);    # missing backend
+$t->json_is('/error', 'Missing parameter: backend');
 $res = $t->post_ok('/api/v1/machines', form => {backend => "kde/usb"})->status_is(400);        #missing name
```

Comment:
`$res` seems unused everywhere. And you can make a method chain like `$t->post_ok(...)->status_is(...)->json_is(...)`. That usually looks better and shows that the tests belong together.

Reply:
I changed the code to use method chain. About $res, it is a problem left over from history, I think if it is necessary it should be fixed in another PR.